tests/xtro-sharpie/common-PDFKit.ignore

```diff
@@ -1,2 +1,9 @@
 # Field used in an EventArg based structure that are in documentation but not in headers
 !unknown-field! PDFAnnotationHit bound
+
+# Change set for XAMCORE_5_0
+!extra-designated-initializer! PDFPage::initWithImage: is incorrectly decorated with an [DesignatedInitializer] attribute
+
+# Uncomment these changes after Xcode 14 UIKit is bound - https://github.com/xamarin/xamarin-macios/issues/15623
+!missing-protocol-conformance! PDFView should conform to UIFindInteractionDelegate
+!missing-selector! PDFView::findInteraction not bound
```

Comment:
This is a bad idea. Leave them in the TODO, that way we do know they have to be done. Else we could forget because people do not look into the ignores for this kind of changes.

Reply:
Agreed